backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -345,6 +333,68 @@ describe('notifications', () => {
               }),
             ).resolves.toEqual(expected)
           })
+
+          describe('if the notification was marked as read earlier', () => {
+            const markAsReadAction = async () => {
+              const mutation = gql`
+                mutation($id: ID!) {
+                  markAsRead(id: $id) {
+                    read
+                  }
+                }
+              `
+              await mutate({ mutation, variables: { id: 'p47' } })
+            }
+
+            describe('but the next mention happens after the notification was marked as read', () => {
+              it('sets the `read` attribute to false again', async () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:08:40Z" title="Friday, August 30th 2019, 2:08:40 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -345,6 +334,68 @@ describe('notifications', () => {
               }),
             ).resolves.toEqual(expected)
           })
+
+          describe('if the notification was marked as read earlier', () => {
+            const markAsReadAction = async () => {
+              const mutation = gql`
+                mutation($id: ID!) {
+                  markAsRead(id: $id) {
+                    read
+                  }
+                }
+              `
+              await mutate({ mutation, variables: { id: 'p47' } })
+            }
+
+            describe('but the next mentioning happens after the notification was marked as read', () => {
```


please don't worry about fixing this now, but so you know it's not really correct to use `mentioning` here... it would be `mention`